dali/operators/image/crop/crop_mirror_normalize.h

```diff
@@ -114,6 +114,26 @@ class CropMirrorNormalize : public Operator<Backend>, protected CropAttr {
     for (auto &element : inv_std_vec_) {
       element = 1.f / element;
     }
+
+    // Handle irregular mean/std argument lengths
+    auto args_size = std::max(mean_vec_.size(), inv_std_vec_.size());
+    if (args_size > 0 && mean_vec_.size() != inv_std_vec_.size()) {
+      if (mean_vec_.empty()) {
+        mean_vec_.push_back(0.0f);  // default
```

Comment:
What if `mean_vec.size() == 0` and `inv_std_vec_.size() > 1` ?
Shouldn't you do:
```
mean_vec_.resize(std::max(1, args_size), 0.0f);
```

Reply:
This can also be achieved by removing `else`.

Reply:
Actually the size can't (and should not) be 0 (since there is a default value). I updated the logic